packages/@react-types/buttongroup/src/index.d.ts

```diff
@@ -15,18 +15,19 @@ import {Alignment, DOMProps, Orientation, StyleProps} from '@react-types/shared'
 import {ReactNode} from 'react';
 
 export interface SpectrumButtonGroupProps extends DOMProps, StyleProps {
-  /** Whether the ButtonGroup is disabled. */
+  /** Whether the Buttons in the ButtonGroup are all disabled. */
   isDisabled?: boolean,
   /**
    * The axis the ButtonGroup should align with. See the docs[LINK TODO] for additional information
-   * regarding overflow behavior.
+   * regarding overflow behavior. Setting this to 'vertical' will prevent any switching behaviors
+   * between 'vertical' and 'horizontal'.
    * @default "horizontal"
    */
   orientation?: Orientation,
   /** The buttons contained within the ButtonGroup. */
   children: ReactNode,
   /**
-   * The alignment of the buttons within the ButtonGroup. Only applicable to "vertical" orientation.  
+   * The alignment of the buttons within the ButtonGroup. Only applicable to "vertical" orientation.
    * @default "start"
    */
   align?: Alignment
```

Comment:
Also, the popover of `Alignment` shows `start` and `end`, but these examples show `center`.